modules/draco/src/lib/draco-parser.js

```diff
@@ -44,6 +45,7 @@ export default class DracoParser {
 
   // NOTE: caller must call `destroyGeometry` on the return value after using it
   parseSync(arrayBuffer, options = {}) {
+    this.metadataQuerier = new this.draco.MetadataQuerier();
```

Comment:
Can we create this object in same place as other Draco objects?

Reply:
We will have to pass it through all methods as argument then.